examples/README.md

```diff
@@ -3,15 +3,14 @@ oneAPI Math Kernel Library (oneMKL) Interfaces offers examples with the followin
 - blas: level3/gemm_usm  
 - rng: uniform_usm  
 - lapack: getrs_usm
-- dft: complex_fwd_buffer, real_fwd_usm
+- dft: complex_fwd_usm, real_fwd_usm
```

Comment:
Any specific reasons for switching from SYCL Buffer APU to USM?

Reply:
Thank for your review @mkrainiuk 
The change is just for consistency with other domains